### ISSUE TITLE
CKEditor not compatible with Screen reader Supernova by Dolphin

### ISSUE THREAD

**Issue by KarelJanVanHaute** [ckeditor/ckeditor5#18389]
## 📝 Provide detailed reproduction steps (if any)
We are getting reports from users that are using the Supernova screen reader that the text fields are just not visible with their screen reader.  
As an example. The demo on this url https://ckeditor.com/ckeditor-5/demo/feature-rich/ is not accessible. 

### ✔️ Expected result

There should be support for screen readers. 

### ❌ Actual result

Every textarea with the CKEditor on are just invisible to the screen reader. 

## 📃 Other details

* Browser: Chrome
* OS: Windows with Supernova by Dolphin
* First affected CKEditor version: 5

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.

**Comment by Reinmar** [comment]
Thanks for letting us know. We do however test CKEditor with a couple of screen readers and none has this issue. Which would indicate that the problem is on the Supernova's side.

cc @Comandeer, do you have any experience with Supernova?

**Comment by Comandeer** [comment]
Unfortunately, not. I'll have to test it.

**Comment by CKEditorBot** [comment]
There has been no activity on this issue for the past year. We've marked it as stale and will close it in 30 days. We understand it may still be relevant, so if you're interested in the solution, leave a comment or reaction under this issue.

**Comment by CKEditorBot** [comment]
We've closed your issue due to inactivity. We understand that the issue may still be relevant. If so, feel free to open a new one (and link this issue to it).
